lib/cli.mjs

```diff
@@ -468,8 +471,10 @@ export default async function main(argv) {
 		options.cacheStrategy = cacheStrategy;
 	}
 
-	if (isBoolean(fix)) {
-		options.fix = fix;
+	if (fix === true || fix === 'lax') {
+		options.fix = 'lax';
+	} else if (fix === 'strict') {
+		options.fix = 'strict';
 	}
```

Comment:
note
minor

If you were to pass `stylelint._options.fix` to `normalizeFixMode` instead of `stylelint._options` you could refactor this part into

```suggestion
	options.fix = normalizeFixMode(fix);`
```

It may complicate the subsequent PR that will support `config.fix`.
@ybiquitous what do you think?

Reply:
Agree. Can you please refactor it?